packages/compass-components/src/components/modal-popover.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { transparentize } from 'polished';
+import { spacing } from '@leafygreen-ui/tokens';
+import { uiColors } from '@leafygreen-ui/palette';
+import { Theme, useTheme } from '../hooks/use-theme';
+import { InteractivePopover } from '..';
+import { gray8 } from '../compass-ui-colors';
+
+const borderRadius = spacing[2];
+
+const contentContainerStyles = css({
+  display: 'flex',
+  height: '100%',
+  alignItems: 'center',
+  borderRadius: borderRadius,
+  boxShadow: `0px 2px 4px -1px ${transparentize(0.85, uiColors.black)}`,
```

Comment:
got this from LG, i'm assuming is what is used to make sure things will keep looking consistent through changes in the palette